tests/python/pytorch/graphbolt/impl/test_in_subgraph_sampler.py

```diff
@@ -156,7 +158,9 @@ def test_InSubgraphSampler_node_pairs_hetero():
     batch_size = 2
     item_sampler = gb.ItemSampler(item_set, batch_size=batch_size)
 
-    in_subgraph_sampler = gb.InSubgraphSampler(item_sampler, graph)
+    in_subgraph_sampler = gb.InSubgraphSampler(
+        item_sampler, graph, output_cscformat=False
```

Comment:
is there any dedicated testcase for `output_cscfomat=True` in this file?

Reply:
Yes. The testcase for csc is in `test_InSubgraphSampler_homo` and `test_InSubgraphSampler_hetero` in this file.